docs/spec/sanitations.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
```

Comment:
Here, the topic should be similar to "Sanitations for OpenAPI specification"
And there should be meta information about authors, reviewers, etc.